docs/readmes/orc8r/override_values_of_terraform_files.md

```diff
@@ -0,0 +1,62 @@
+---
+id: override_values_of_terraform_files
+title: Override values of Terraform files
+hide_title: true
+---
+# Override values of Terraform files
+
+**Description:** This guide drescribed how to override values that are part of the terraform files that are used in the github repository.
```

Comment:
Can we change this to `This guide describes how to override values ....`

Reply:
Thank you! I have updated